### PR TITLE
add-wallet-test

### PR DESCRIPTION
**Pull request by ssallam**

### CODE REVIEW

File: tests/web3_internal/test_wallet.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    # Create wallet with valid private key
+    wallet = Wallet(web3, private_key=private_key)
+    assert wallet.private_key == private_key
+    assert wallet.address
+    signed_message = wallet.sign(add_ethereum_prefix_and_hash_msg("msg-to-sign"))
+    assert signed_message
+
+    # Create wallet with encrypted key and password
+    password = "darksecret"
```
Comment (mariacarmina) [inline tests/web3_internal/test_wallet.py:24]: Codacy cries about the hardcoded password but in this context I think we can't do anything about that.

Comment (calina-c) [reply tests/web3_internal/test_wallet.py:24]: agreeed.

**Review by calina-c — APPROVED** [review]

**Review by mariacarmina — APPROVED** [review]
Maybe a re-run should be fine for `Travis Branch` to pass.

**Comment by calina-c** [comment]
Merging, the one Codacy issue left is irrelevant.

**Review by calina-c — APPROVED** [review]

**Pull request merged**